packages/modules/redis/src/redis-container.ts

```diff
@@ -0,0 +1,73 @@
+import { AbstractStartedContainer, GenericContainer, StartedTestContainer } from "testcontainers";
+
+const REDIS_PORT = 6379;
+
+export class RedisContainer extends GenericContainer {
+  private password = "";
+  private persistenceVolume = "";
```

Comment:
Let's use an optional type `private persistenceVolume?: string` and then check `if (this.persistenceVolume)` to get type-checking support